himan-lib/include/grid.h

```diff
@@ -210,8 +217,19 @@ class regular_grid : public grid
 
 	/* Return grid point value (incl. fractions) of a given latlon point */
 	virtual point XY(const point& latlon) const;
+
+	/* Return grid point value (location) of a given target grid */
+	virtual std::vector<point> XY(const regular_grid& to) const;
+
+	/* Return latitude-longitude value for a given location index */
```

Comment:
I personally wouldn't define a bunch of new functions for the bulk calculations but use the single point functions as unary operators in std:transform algorithm library function.

Reply:
The bulk function is using a slightly different method to transform coordinates, hence it cannot be done the way you describe.